avm/res/cognitive-services/account/tests/e2e/ai-model-deployment/main.test.bicep

```diff
@@ -0,0 +1,63 @@
+targetScope = 'subscription'
+
+metadata name = 'Using deployments parameter set'
+metadata description = 'This instance deploys the module with ai model deployment feature.'
```

Comment:
```suggestion
metadata name = 'Using `deployments` in parameter set'
metadata description = 'This instance deploys the module with the AI model deployment feature.'
```

Reply:
Please run the `Set-AVMModule` script one last time after this update.

Reply:
Updated!

Reply:
Much appreciated. All good from my side. Thanks